.github/pull_request_template.md

```diff
@@ -4,6 +4,7 @@ Fixes https://xenitsupport.jira.com/browse/ALFREDAPI-<**YOUR TICKET ID**>
 - [ ] Does this PR avoid breaking the API? 
     Breaking changes include adding, changing or removing endpoints and/or JSON objects used in requests and responses.
 - [ ] Does the PR comply to REST HTTP result codes policy outlined in the [developer guide](https://github.com/xenit-eu/alfred-api/blob/master/developer-documentation)?
+- [ ] Is errorhandling done through a method annotated with `@ExcpetionHandler`in the webscript class?
```

Comment:
- errorhandling -> error handling
- ExcpetionHandler-> ExceptionHandler
- `@ExcpetionHandler`in -> `@ExceptionHandler` in (space missing)

Also, can you put webscript class**es** at the end?

Reply:
Also weird build error. I don't think it has anything to do with this PR, so try rebuilding.